.gitignore

```diff
@@ -1,8 +1,37 @@
-# ignored files
-*.tfstate
-*.tfstate.backup
-.terraform
 .idea
-*.iml
-go.sum
-*.terraform.lock.hcl
\ No newline at end of file
+
+#################################################################
+# Default .gitignore content for all terraform-aws-modules below
+#################################################################
+
+# Local .terraform directories
+**/.terraform/*
+
+# Terraform lockfile
+**/*.terraform.lock.hcl
+
+# .tfstate files
+**/*.tfstate
+**/*.tfstate.*
+**/*.tfplan
+
+# Crash log files
+crash.log
+**/*.zip
```

Comment:
please remove this `**/*.zip` as we need zip file for lambda in `_example/complete` directory.

Reply:
Done